org.zrythm.Zrythm.json

```diff
@@ -0,0 +1,468 @@
+                    "url": "http://code.breakfastquay.com/attachments/download/34/rubberband-1.8.1.tar.bz2",
+                    "sha256": "ff0c63b0b5ce41f937a8a3bc560f27918c5fe0b90c6bc1cb70829b86ada82b75"
+                }
+            ],
+            "post-install": [
+                "install -Dm644 -t /app/share/licenses/rubberband COPYING"
+            ],
+            "cleanup": [
+                "/bin",
+                "/include",
+                "/share/ladspa",
+                "*.a",
+                "*.so"
+            ]
+        },
+        {
+            "name": "reproc",
+            "buildsystem": "cmake-ninja",
+            "build-options": {
+                "env": {
+                    "BUILD_SHARED_LIBS": "ON",
+                    "REPROC_WARNINGS": "ON",
+                    "REPROC_WARNINGS_AS_ERRORS": "ON",
+                    "CFLAGS": "-fPIC"
+                }
+            },
+            "sources": [
+                {
+                    "type": "git",
+                    "url": "https://github.com/DaanDeMeyer/reproc",
+                    "commit": "8aea3aef64c37a7ed0db759a7128a80c2540c808",
+                    "tag": "v14.2.4"
+                }
+            ]
+        },
+        {
+            "name": "breeze-icons",
+            "builddir": true,
+            "buildsystem": "simple",
+            "build-commands": [
+                "mkdir -p /app/share/icons/breeze-dark",
+                "cp -r icons-dark/* /app/share/icons/breeze-dark"
+            ],
+            "sources": [
+                {
+                    "type": "git",
+                    "url": "https://github.com/KDE/breeze-icons",
+                    "commit": "c8ca52623d49f77fe32dc1bdb400fa16bc4afb06",
+                    "tag": "v5.90.0"
+                }
+            ]
+        },
+        {
+            "name": "lsp-dsp-lib",
+            "buildsystem": "simple",
+            "build-commands": [
+                "make PREFIX=/app config",
+                "make PREFIX=/app -j $FLATPAK_BUILDER_N_JOBS",
+                "make PREFIX=/app install"
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://github.com/lsp-plugins/lsp-dsp-lib/releases/download/0.5.14/lsp-dsp-lib-0.5.14-src.tar.gz",
+                    "sha256": "781dae54c934bc11c412a7abbea1321e4ea6513ca24327627b8bfbe5fdb49fbd"
+                }
+            ]
+        },
+        {
+            "name": "pipewire",
+            "buildsystem": "meson",
+            "config-opts": [
+                "-Dgstreamer=disabled",
+                "-Dman=disabled",
+                "-Dsystemd=disabled",
+                "-Dudev=disabled",
+                "-Dudevrulesdir=disabled",
+                "-Dsession-managers=['']"
+            ],
+            "sources": [
+                {
+                    "type": "git",
+                    "url": "https://github.com/pipewire/pipewire.git",
```

Comment:
why do you need this?

Reply:
https://gitlab.freedesktop.org/pipewire/pipewire/-/issues/1591

Reply:
actually, 0.3.43 has this bug as well. using 0.3.44 (just released today) would be better since it includes a fix